### PR TITLE
build prototype of RegisterFindSleepSpecialist page

### PR DESCRIPTION
**Pull request by martindev727**

### CODE REVIEW

File: src/components/register-find-sleep-specialist/register-find-sleep-specialist.component.jsx [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+            </p>
+            <p className="form-comment">
+              If you are already in the directory and wish to update or remove your listing, <SiteLink className="link-text" to="Locate Sleep Specialist">click here</SiteLink>.
+              If you are not a healthcare professional, please <SiteLink className="link-text" to="Locate Sleep Specialist">click here</SiteLink>.
+            </p>
+          </div>
+        </div>
+        <div className="primary-container">
+          <form>
+            <div className="form-row">
```
Comment (rphovley) [inline src/components/register-find-sleep-specialist/register-find-sleep-specialist.component.jsx]: screen shot 2018-08-30 at 11 15 37 am

We need to do some touching up on the sizes between full screen and mobile. Potentially putting each line on full screen becoming 2 lines or something like that. Also it appears that some of the bottom borders are getting covered up at some sizes

File: src/components/register-find-sleep-specialist/register-find-sleep-specialist.component.jsx [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+              </div>
+            </div>
+            <div className="form-row last-count">
+              <p className="item-comment">
+                How many patients have you diagnosed with narcolepsy in the past 24 months?*
+              </p>
+              <FloatingInputLabel id="lastCount" className="input-small" shrink={60} label="" onChange={(val) => this.handleInputChange('lastCount', val)} />
+            </div>
+            <div className="form-row checkbox-group">
+              <CustomizeCheckboxComponent checkboxContent = {checkbox1} onChange={(e) => this.handleInputChange('isCertify', e.target.checked)} />
```
Comment (rphovley) [inline src/components/register-find-sleep-specialist/register-find-sleep-specialist.component.jsx]: screen shot 2018-08-30 at 11 27 12 am
This is what the checkboxes look on the production site. They will need to be modified to look like this instead

File: src/components/register-find-sleep-specialist/register-find-sleep-specialist.component.jsx [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+            </div>
+            <div className="form-row">
+              <FloatingInputLabel id="npiNumber" className="input-normal" shrink={60} label="NPI Number" onChange={(val) => this.handleInputChange('npiNumber', val)} />
+            </div>
+            <div className="form-row option-group">
+              <p className="item-comment">
+                Do you treat pediatric patients?*
+              </p>
+              <div>
+                <CustomizeOptionComponent optionContent={option1} onChange={(e) => this.handleInputChange('isPatients', e.target.checked)} />
```
Comment (rphovley) [inline src/components/register-find-sleep-specialist/register-find-sleep-specialist.component.jsx]: Take the option component that is being used for the symptom screener and use that one instead `/screen/symptom-screener/`

File: src/components/register-find-sleep-specialist/register-find-sleep-specialist.component.jsx [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+]
+const checkbox1 = {
+  id: 'checkbox1',
+  label: 'I certify that I have been trained to diagnose and differentiate sleep disorders.',
+  required: true,
+  className: 'checkbox-item',
+}
+const checkbox2 = {
+  id: 'checkbox2',
+  label: 'I accept the Privacy Policy and Terms of Use*',
```
Comment (rphovley) [inline src/components/register-find-sleep-specialist/register-find-sleep-specialist.component.jsx]: There needs to be a link on "Privacy Policy and Terms of Use*" that takes them to `/privacy-statement/`

File: src/components/register-find-sleep-specialist/register-find-sleep-specialist.component.jsx (hunk 2 above)
Comment (martindev727) [reply src/components/register-find-sleep-specialist/register-find-sleep-specialist.component.jsx]: Fixed

File: src/components/register-find-sleep-specialist/register-find-sleep-specialist.component.jsx (hunk 3 above)
Comment (martindev727) [reply src/components/register-find-sleep-specialist/register-find-sleep-specialist.component.jsx]: Fixed

**Comment by rphovley** [comment]
Also, the text on inputs needs to match designs

I believe everything is good except for text color.
screen shot 2018-09-04 at 8 30 46 am

File: src/components/register-find-sleep-specialist/register-find-sleep-specialist.component.jsx [hunk 5]
```diff
@@ -0,0 +1,268 @@
...
+              <p className="item-comment">
+                How many patients have you diagnosed with narcolepsy in the past 24 months?*
+              </p>
+              <FloatingInputLabel id="lastCount" className="input-small" shrink={60} label="" onChange={(val) => this.handleInputChange('lastCount', val)} />
+            </div>
+            <div className="form-row checkbox-group">
+              <CustomizeCheckboxComponent checkboxContent = {checkbox1} onChange={(e) => this.handleInputChange('isCertify', e.target.checked)} />
+              <CustomizeCheckboxComponent checkboxContent = {checkbox2} onChange={(e) => this.handleInputChange('isAccept', e.target.checked)} />
+            </div>
+            <div className="form-row button-group">
```
[the remark this answers is not in the document]
Comment (martindev727) [reply src/components/register-find-sleep-specialist/register-find-sleep-specialist.component.jsx]: Fixed

File: src/components/register-find-sleep-specialist/register-find-sleep-specialist.component.jsx [hunk 6]
```diff
@@ -0,0 +1,268 @@
...
+              In order to update your profile in the Sleep Specialist directory, you must confirm your identity by entering your State License
+              and DEA numbers. (This information is private and will not be shared with anyone, including website users.)
+            </p>
+            <div className="form-row confirm-identity">
+              <div className="input-group">
+                <FloatingInputLabel id="stateLicenseNumber" className="input-normal" shrink={60} label="State License Number*" onChange={(val) => this.handleInputChange('stateLicenseNumber', val)} />
+                <FloatingInputLabel id="licenseExpiredDate" className="input-normal" shrink={60} label="License Expiration Date*" onChange={(val) => this.handleInputChange('licenseExpiredDate', val)} />
+              </div>
+              <div className="react-floating-label-input input-medium">
+                <select className="form-select"
```
[the remark this answers is not in the document]
Comment (martindev727) [reply src/components/register-find-sleep-specialist/register-find-sleep-specialist.component.jsx]: Fixed select box arrow color

File: src/components/register-find-sleep-specialist/register-find-sleep-specialist.component.jsx [hunk 7]
```diff
@@ -0,0 +1,277 @@
...
+                Do you treat pediatric patients?*
+              </p>
+              <div className="radio-patient-wrapper">
+                <div className="option-patient-section">
+                  <div className="radio yes">
+                    <label className="option-label">Yes</label>
+                    <input className="option-radio" type="radio" value="option1" name="gender" id="yes-box" checked onChange={(e) => this.handleInputChange('isPatient', !e.target.value)} />
+                    <label htmlFor="yes-box" />
+                  </div>
+                  <div className="radio no">
```
Comment (rphovley) [inline src/components/register-find-sleep-specialist/register-find-sleep-specialist.component.jsx]: If you select no and then check one of the two checkboxes, the option resets to "yes"

**Comment by martindev727** [comment]
Radio validation fixed.

**Comment by rphovley** [comment]
Between the pixel width of 670 and 700 or so the submit buttons are positioned oddly
screen shot 2018-09-05 at 11 03 25 am

**Pull request closed without merging**